### PR TITLE
docs(tabs): onTabClick description

### PR DESCRIPTION
**Pull request by akaguny**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

to small informaion about tabs api

### 💡 Solution

* onTabClick collback description provided by rc-tabs
* link to rc-tabs docs (for read more)

### 📝 Changelog description

> Describe changes from userside, and list all potential break changes or other risks.

1. English description

2. Chinese description (optional)

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/tabs/index.en-US.md [hunk 1]
```diff
@@ -45,3 +46,4 @@ Ant Design has 3 types of Tabs for different situations.
 | forceRender | Forced render of content in tabs, not lazy render after clicking on tabs | boolean | false |
 | key | TabPane's key | string | - |
 | tab | Show text in TabPane's head | string\|ReactNode | - |
+More option at [rc-tabs option](https://github.com/react-component/tabs#tabpane)
```
Comment (afc163) [inline components/tabs/index.en-US.md]: Would you mind adding a blank line before this?

Comment (akaguny) [reply components/tabs/index.en-US.md]: no need to add blank line after table https://deploy-preview-15420--ant-design.netlify.com/components/tabs/#header

Comment (afc163) [reply components/tabs/index.en-US.md]: We should keep markdown source clear and readable too.

Comment (akaguny) [reply components/tabs/index.en-US.md]: In the future maybe prettier for this?

**Pull request merged**